packages/in_app_purchase/lib/src/in_app_purchase_connection/app_store_connection.dart

```diff
@@ -26,4 +27,26 @@ class AppStoreConnection implements InAppPurchaseConnection {
     _instance = AppStoreConnection();
     return _instance;
   }
+
+  /// Query the product detail list.
+  ///
+  /// This method only returns [QueryProductDetailsResponse].
```

Comment:
FYI I don't think this or `GooglePlayConnection` really need dart doc, since we're not exposing either of them as public API classes and they're overriding the documented public interface. You also don't need to delete documentation you've already written though.